src/gfx_d3d12.zig

```diff
@@ -1188,6 +1229,8 @@ pub fn endFrame(state: *D3D12State, camera: *const fd.Camera, camera_position: [
     zpix.endEvent(gctx.cmdlist); // End GBuffer event
 
     const ibl_textures = state.lookupIBLTextures();
+    const point_lights_buffer = state.lookupBuffer(state.point_lights_buffers[gctx.frame_index]);
+    const num_point_lights = state.num_point_lights[gctx.frame_index];
```

Comment:
Minor nitpick but I tend to not use `num_xyz`. Putting it at the end means they autocomplete nicely and show up in the debugger aligned with each other. So I would name it `point_lights_count`.

Reply:
oh I see. No worries, I'll rename it (and max_num_lights)